src/components/views/styl/DataTable.styl

```diff
@@ -1,75 +1,63 @@
+$table-vertical-padding = 15px;
+
 .data-table-wrapper {
     width: 100%;
 }
 
 .data-table {
     border-collapse: collapse;
     width: 100%;
-}
-
-.data-table thead tr th {
-    border-bottom: 1px solid #cfcfcf;
-}
-
-.data-table thead th {
-    padding-bottom: 15px; /* Matches the value of rule with this keyword: header-space */
-}
-
-.data-table td {
-    padding: 8px 0;
-}
-
-.data-table tr:first-child td {
-    padding-top: 15px; /* Matches the value of rule with this keyword: header-space */
-}
-
-.data-table th:first-child, .data-table td:first-child {
-    text-align: left;
-}
-
-.data-table td:first-child {
-    padding-left: 10px;
-}
-
-.data-table th:nth-child(2), .data-table td:nth-child(2) {
-    text-align: left;
-}
-
-.data-table th:last-child, .data-table td:last-child {
-    text-align: right;
-    padding-right: 10px;
-}
-
-/* Give the columns fixed widths, otherwise their widths might change when
-   values change.
-
-   These should all add up to the total width of the table. They need to be
-   fixed widhts because otherwise truncation (see below) wouldn't work. */
-.data-table .rank {
-    width: 50px;
-}
-
-.data-table .name {
-    /* Width set below */
-}
-
-.data-table .value {
-    width: 220px;
-}
-
-/* Truncate long add-on names. These properties can't be set on a selector that
-   targets a <col>, so we need to select the <td> directly.
-
-   https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/ */
-.data-table td:nth-child(2) {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-
-    /* Needs to be max-width or else truncation won't work */
-    max-width: 145px;
-}
 
-.data-table tr:nth-child(even) td {
-    background-color: #efefef;
+    thead {
+        tr th {
+            border-bottom: 1px solid #cfcfcf;
+        }
+        th {
+            padding-bottom: $table-vertical-padding; // Matches the value of rule with this keyword: header-space
```

Comment:
I don't think we need that comment any more. I just used these comments because we didn't have variables. :smiley: